src/platform_impl/windows/window.rs

```diff
@@ -882,6 +882,11 @@ unsafe fn post_init<T: 'static>(
         thread_executor: event_loop.create_thread_executor(),
     };
 
+    // Set visible before setting the size
+    // to ensure the the attribute is correctly
+    // applied.
```

Comment:
nit: This feels kind of cramped.
```suggestion
    // Set visible before setting the size to ensure the
    // attribute is correctly applied.
```
It could also be on one line, IMO.